### DISCUSSION TITLE
Question: How to use IntendedFor field in bidscoin correctly

### DISCUSSION THREAD

**Discussion by thomshaw92** [Donders-Institute/bidscoin#168] [Q&A]
I've had a quick read through the documentation for both BIDS and BidsCoin, but I can't find a clear answer on what to put in the IntendedFor field. 
I think I'm having a little bit of trouble understanding this part of the docs:

"For instance you can use task to select all functional runs or use <<Stop*Go><Reward>> to select “Stop1Go”-, “Stop2Go”- and “Reward”-runs. NB: bidsapps may not use your fieldmap at all if you leave this field empty!"

So, for example, we have a B1 fieldmap for correcting our T1w scan - does that mean I would put 
```
<<*T1w*>>
```
in the IntendedFor field? 
Thanks for your help!

**Comment by marcelzwiers** [comment]
Yes, indeed, the part between the `<<` and `>>` is used as a search term to find other bids-files in the sub-#/ses-# folder. So you could even include the underscore: `<<_T1w>>` to be just a bit more selective

**Comment by marcelzwiers** [comment]
p.s. the search term is automatically padded with `*` at the end and start, so you only need them in the middle, e.g. `<<run-2*_T1w>>`

**Comment by thomshaw92** [comment]
Thanks for the help :)

**Comment by Michael-Monaldi** [comment]
We are unable to get the IntendedFor field to populate on fmaps.
This happens on 3.6.3 stable and on 3.7.0dev
I have used the following in the bidsmap_dccn.yaml

bold
\*bold*
\<bold>
\<\<bold>>
\<\*bold*>
\<\<\*bold*>>

All the above variants, using the word "task"
\<\<task-*bold>>

The IntendedFor ends up being either:
what was literally typed including the "< and>" characters, or
IntendedFor : null

**Reply by marcelzwiers** [reply]
Sorry for the late reply, but what do you mean with "end up" exactly? Ending up in the bidsmap or in the json-files? The idea is that the `IntendedFor` remains what it is in the bidsmap, and that the plugin uses that info to search for matching images in that session. Those matches (if any) are then written to the json sidecar files.
